app/webpacker/css/darkswarm/mixins.scss

```diff
@@ -213,6 +213,5 @@
 }
 
 @mixin force-wrap {
-  white-space: pre-wrap;
```

Comment:
You could have removed this detour before submitting the PR. :wink:

Reply:
Right. I've force-pushed.